src/setup.ts

```diff
@@ -1,27 +1,28 @@
+import {getOctokit} from "@actions/github";
 import * as tc from "@actions/tool-cache";
 import * as core from "@actions/core";
-import fetch from "node-fetch"
 
-export async function getLatestVersion() {
+type Release = {
+  tag_name: string;
+};
+
+export async function getLatestVersion(token: string) {
   const url = "https://api.github.com/repos/loilo-inc/canarycage/releases";
-  type Release = {
-    tag_name: string;
-  };
-  const res = await fetch(url)
-  if (res.status === 200) {
-    const list: Release[] = await res.json()
+  const res = await getOctokit(token).request(`GET ${url}`);
```

Comment:
Please use the canonical way.
```suggestion
    const res = await getOctokit(token).rest.repos.listReleases({ owner: "loilo-inc", repo: "canarycage" })
```

Reply:
@keroxp [b1a82da](https://github.com/loilo-inc/actions-setup-cage/pull/20/commits/b1a82dae21ead7b45855c8b4091dd0deb1955ca5) fixed